construct.yaml

```diff
@@ -51,12 +51,10 @@ specs:
   - boto3
   - gfal2-util
   - fts3 >=0.0.0.1.dev2
+  - htcondor-utils  # [linux64]
```

Comment:
Also I see that GSI will be entriely removed from htcondor pretty soon: https://htcondor-wiki.cs.wisc.edu/index.cgi/wiki/index.cgi/wiki?p=PlanToReplaceGridCommunityToolkit

Conda-forge is [currently building the `9.0.x` series releases](https://github.com/conda-forge/htcondor-feedstock/blob/master/conda-forge.yml#L1-L3) in addition to the lastest (proably for LIGO) so we could stick to that one for while this settles down if preferred?